spec/ScorecardSpec.js

```diff
@@ -0,0 +1,158 @@
+'use strict';
+
+describe('Scorecard', function() {
+  var scorecard;
+
+  beforeEach(function() {
+    scorecard = new Scorecard();
+  });
+
+  it('starts with empty scorecard', function() {
```

Comment:
You could scoop your code in another `describe`, I can see you are testing for `getFrames()`, you could have all those tests for this method in an another describe block `describe("getFrames()",()=>{all the it here})`, and do the same for the repetitive methods.